Fastfile

```diff
@@ -74,6 +92,7 @@ default_platform :ios
   end
 
   after_all do |lane|
+    notifiy_slack()
```

Comment:
Is it worth thinking about that if `after_all` is implemented in the user's fastfile after the import of the master fastfile this gets overridden? I found this out recently using the master fastfile for something at home